### PR TITLE
v10.1.0

### PR DESCRIPTION
**Pull request by estebanmino**
### Added
- Export BaseControllerV2 Json type ([#482](https://github.com/MetaMask/controllers/pull/482))

### Changed
- Improve collectible detection by account ([#487](https://github.com/MetaMask/controllers/pull/487))
- Upgrade ethereumjs util ([#466](https://github.com/MetaMask/controllers/pull/466))

### Fixed
- Skip token detection for tokens that are already tracked ([#480](https://github.com/MetaMask/controllers/pull/480))

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [10.1.0] - 2021-06-7
+### Added
+- Export BaseControllerV2 Json type ([#482](https://github.com/MetaMask/controllers/pull/482))
+
+### Changed
+- Improve collectible detection by account ([#487](https://github.com/MetaMask/controllers/pull/487))
+- Mapping token address strings instead of Token objects while checking for new tokens ([#480](https://github.com/MetaMask/controllers/pull/480))
```
Comment (Gudahtt) [inline CHANGELOG.md:15]: This PR was more of a bug fix, or at least that was my understanding. I would put this under Fixed, with the description "Skip token detection for tokens that are already tracked", or something like that. That was the bug - that we're detecting _all_ tokens, even those already on the user's list.

**Review by Gudahtt — DISMISSED** [review]
LGTM! Left a suggestion for the changelog, but we can adjust that in a follow-up if you're in a rush to release this.

**Comment by estebanmino** [comment]
@Gudahtt changed tha changelog as you suggested! 👌

**Review by Gudahtt — APPROVED** [review]
Thanks!

**Comment by Gudahtt** [comment]
>A new version of the package @metamask/controllers (10.1.0) was published at 2021-06-07T16:08:13.739Z from
[redacted]. The shasum of this package was ec0a3751fa658966e9818038784ab6b555c95382.

**Pull request merged**